### ISSUE TITLE
Windows build semi-broken for rc6 with libusb

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nfc-tools/libnfc#238]
```
The updated conditional on libusb didn't quite work for me.
Perhaps its specific to how I do things, but I tend not to "install" SDKs, so 
CMake does not discover libusb in a pre-defined location.

I manually fill the include and lib path, which allows things to run and build.

Thus, there's a small CMakeLists.txt modification that's needed.
I don't see the need to make a branch and explicit change for you to pull in, 
because it's pretty simple:

In /CMakeLists.txt (root directory)
Line 122, add:
SET(LIBUSB_FOUND TRUE)

Basically, forcing the LIBUSB_FOUND variable on by the act of defining the 
directories.
```

Original issue reported on code.google.com by `alex.l...@gmail.com` on 8 Mar 2013 at 6:24
